File: trunk/Templates/Examples/Xml/output/Employee.cs
using System;

namespace Northwind.DAL
{
	/// <summary>
	/// This object represents the properties and methods of a Employee.
	/// </summary>
	public class Employee
	{
		private Int32 _employeeID;
		private String _lastName;
		private String _firstName;
		private String _title;
		private String _titleOfCourtesy;
		private DateTime _birthDate;
		private DateTime _hireDate;
		private String _address;
		private String _city;
		private String _region;
		private String _postalCode;
		private String _country;
		private String _homePhone;
		private String _extension;
		private Byte[] _photo;
		private String _notes;
		private Int32 _reportsTo;
		private String _photoPath;
		
		public Employee()
		{
		}
		
		#region Custom - Methods
		// Insert custom methods in here so that they are preserved during re-generation.
		#endregion
	}
}

File: branches/5.x/Templates/Examples/Xml/VisualBasic/output/Employee.vb
Imports System

Namespace Northwind.DAL
	''' <summary>
	''' This object represents the properties and methods of a Employee.
	''' </summary>
	Public Class Employee
		Private _employeeID As Int32
		Private _lastName As String
		Private _firstName As String
		Private _title As String
		Private _titleOfCourtesy As String
		Private _birthDate As DateTime
		Private _hireDate As DateTime
		Private _address As String
		Private _city As String
		Private _region As String
		Private _postalCode As String
		Private _country As String
		Private _homePhone As String
		Private _extension As String
		Private _photo As Byte[]
		Private _notes As String
		Private _reportsTo As Int32
		Private _photoPath As String

		Public Sub New()
		End Sub

		#region Custom - Methods
		' Insert custom methods in here so that they are preserved during re-generation.
#End Region
	End Class
End Namespace

